.pre-commit-config.yaml

```diff
@@ -13,14 +14,62 @@ repos:
       - id: check-merge-conflict
       # detects the presence of private keys
       - id: detect-private-key
+
   - repo: https://github.com/commitizen-tools/commitizen
     rev: v3.27.0
     hooks:
       - id: commitizen
         name: Check commit message format
+
   - repo: local
     hooks:
       - id: prettier
         name: Check files format with prettier
-        entry: bash -c 'prettier --write **/*'
+        entry: bash -c 'prettier --write .'
         language: system
+
+  # Python SDK hooks
+  - repo: local
+    hooks:
+      # Install Python SDK
+      - id: python-sdk-install
+        name: Install python SDK
+        language: python
+        types: [python]
+        # Custom command for monorepo
+        entry: bash -c 'cd sdks/python && pip install .[dev,test]'
```

Comment:
I have not been able to get this step running successfully.

```
      ...
      error: [Errno 17] File exists: 'build/bdist.macosx-14.4-arm64/wheel/redactive-0.0.5.dist-info'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for redactive
Failed to build redactive
ERROR: Could not build wheels for redactive, which is required to install pyproject.toml-based projects
```

Reply:
hm.. The package install in pre-commit env executes differently. Let me check first.